packages/core/src/App/Containers/RealityCheckModal/reality-check-modal.jsx

```diff
@@ -3,25 +3,50 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { useHistory } from 'react-router-dom';
 import { Input } from '@deriv/components';
-import { routes, validNumber } from '@deriv/shared';
+import { getDecimalPlaces, routes, validNumber } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 import BriefModal from './brief-modal.jsx';
 import SummaryModal from './summary-modal.jsx';
 
-const IntervalField = ({ values, touched, errors, handleChange, handleBlur }) => (
+const SpendingLimitIntervalField = ({ values, disabled, touched, errors, handleChange, handleBlur }) => (
     <div className='reality-check__fieldset'>
         <Field name='interval'>
             {({ field }) => (
                 <Input
                     {...field}
+                    name='max_30day_turnover'
                     data-lpignore='true'
                     type='text'
-                    label={localize('Time interval')}
+                    label={localize('My spending limit (EUR)')}
+                    value={values.max_30day_turnover}
+                    onChange={handleChange}
+                    onBlur={handleBlur}
+                    disabled={disabled}
+                    required
+                    error={touched.max_30day_turnover && errors.max_30day_turnover}
+                    autoComplete='off'
+                    maxLength='13'
+                />
+            )}
+        </Field>
+    </div>
+);
+
+const TradingViewIntervalField = ({ values, touched, errors, handleChange, handleBlur }) => (
+    <div className='reality-check__fieldset'>
+        <Field name='interval'>
+            {({ field }) => (
+                <Input
+                    {...field}
+                    name='interval'
+                    data-lpignore='true'
+                    type='text'
+                    label={localize('After … minutes of trading')}
                     value={values.interval}
                     onChange={handleChange}
                     onBlur={handleBlur}
-                    hint={localize('Interval should be between 10-60 minutes')}
+                    hint={localize('Min: 10 minutes     Max: 60 minutes')}
```

Comment:
Is the whitespace intentional?

Reply:
@zaki-binary yes, it's whitespace

Reply:
Ok, but best to give a heads up to translations team as I think the whitespaces might not necessarily translate well to different languages.

Reply:
@shayan-binary-2 Please help to add a comment here for future references.